packages/salesforcedx-vscode-soql/.vscodeignore

```diff
@@ -12,7 +12,43 @@ tsconfig.json
 *.xml
 **/*[!README].md
 *.vsix
+
+# local rxjs node modules
 node_modules/rxjs/src/**
 node_modules/rxjs/bundles/**
 node_modules/rxjs/_esm5/**
-node_modules/rxjs/_esm2015/**
\ No newline at end of file
+node_modules/rxjs/_esm2015/**
+node_modules/typescript/**
```

Comment:
Why was typescript getting added to the vsix ?

Reply:
I'm not sure. I'd like to spend more time digging into the dependency chain with the work item.  But for now, i'm gonna merge this for todays release.

 I used this to find the biggest directories in node modules.  
`du -a . | sort -n -r | head -n 20`

Reply:
The lwc extension has a similar problem where `typescript` is getting imported as a child dependency via `jest-editor-support` and we can't remove it (yet) since it breaks all the testing capabilities it provides for LWC users. 

You should check if you didn't break anything in the extension by using the vsix, unfortunately the automation won't catch this for you.